src/components/ControlPanel/controlPanel.js

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
 
-
 export default class ControlPanel extends React.Component {
 	render() {
 		return (
 			<div className="control-panel-wrapper">
-				<h1>hello world</h1>
+				<div className="form-elem">
+					<p className="SubheaderSecondary">Committee</p>
+					<select className="Display-2Primary">
```

Comment:
This select currently has the up/down arrows next to it. Change it so that:
- It doesn't have those arrows (see how the year dropdown looks on the member view)
- Font size is much smaller